### DISCUSSION TITLE
What does "Sort title" do?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [bookwyrm-social/bookwyrm#3220] [Q&A] [answered]
https://bookwyrm.social/book/56047/edit#:~:text=Title%3A-,sort%20title%3A%20,-Subtitle%3A

**Comment by mouse-reeve** [comment, marked as answer]
You're not the only person wondering! It's a field for the title minus any grammatical articles, to produce better sorting of search results. So for example, "The Lord of the Rings" should be sorted as if its title starts with "L" instead of "T", so the sort title is "Lord of the Rings"
